draft-ietf-mops-streaming-opcons.md

```diff
@@ -291,15 +293,15 @@ as low as a per-feed requirement instead of a per-user requirement.
 
 ##Caching Systems
 
-TBD: pros, cons, tradeoffs of caching designs at different locations within
-the network?
+When demand for content is relatively predictable, and especially when that content is relatively static, caching content close to requesters, and pre=loading caches to respond quickly to initial requests, is often useful. This is subject to the usual considerations for caching - for example, how much data must be cached to make a significant difference to the requester, and how the benefits of caching and pre-loading caches balances against the costs of tracking "stale" content in caches and refreshing that content.
```

Comment:
nit: pre=loading -> pre-loading

Reply:
Would it make sense to reference maybe RFC 7234 here to follow up on "usual considerations for caching"?  Likewise wondering if there's any good papers analyzing tradeoffs of preloading caches, a 2-minute search fails me.

Reply:
Thanks for these. I pushed an update to make these changes (although RFC 7234 seemed more about the mechanics of caching than the strategy of caching, so I made the reference point to caching itself.